.github/workflows/ue-docker.yml

```diff
@@ -0,0 +1,79 @@
+name: Build UE Windows Docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      ue_version:
+        description: Select Unreal Engine version
+        required: true
+        type: choice
+        options:
+        - 4.27
+        - 5.0
+        - 5.1
+        - 5.2
+        - 5.3
+        - 5.4
+        - 5.5
```

Comment:
Is it worth having options here instead of letting user specify it (which would mean we don't need to change this file when a new version is built?

Reply:
Let's keep the options for now as they make it clear which UE versions are supported and ensure that developers patch only the corresponding Unreal fork branches used for building the Docker images.